src/qililab/platform/platform.py

```diff
@@ -574,7 +580,7 @@ def execute_qprogram(self, qprogram: QProgram) -> dict[str, list[Result]]:
         # Reset instrument settings
         for instrument in self.instruments.elements:
             if isinstance(instrument, QbloxModule):
-                instrument.reset_sequences()
+                instrument.clear_cache()  # FIXME: allow cache with QProgram
```

Comment:
Why is this a `FIXME`? I see this comment more of a new feature rather than a bug

Reply:
Fair enough